### PR TITLE
Feat/404 page

### PR DESCRIPTION
**Pull request by KaiElwood**
Added 404 page changes #84

### CODE REVIEW

**Comment by KaiElwood** [comment]
I accidentally added two unrelated commits so I reverted them back, that's what all that ^ was :)

File: wp-content/themes/csisnuclearnetwork/assets/_scss/pages/404.scss [hunk 1]
```diff
@@ -3,38 +3,65 @@
...
+      padding-top: rem(40);
+    }
   }
 
-  &__title {
-    // color: $color-accent-light-primary-200;
-    // @extend %text-style-label-3-x-large;
+  &__img {
+    margin: auto;
+    @include breakpoint('large') {
```
Comment (urchykli) [inline wp-content/themes/csisnuclearnetwork/assets/_scss/pages/404.scss:32]: This happens at `medium`.

File: wp-content/themes/csisnuclearnetwork/assets/_scss/pages/404.scss [hunk 2]
```diff
@@ -3,38 +3,65 @@
...
+    @extend %text-style-label-large-strong-caps;
+    padding: rem(32) 0 rem(8) 0;
   }
 
   .intro-text {
     p {
-      color: $color-black-100;
-      // @extend %text-style-paragraph-medium-long;
+      @extend %text-style-paragraph-large;
+      @extend %text-style-long;
```
Comment (urchykli) [inline wp-content/themes/csisnuclearnetwork/assets/_scss/pages/404.scss:60]: Please remove this and add the `text--long` class to the element in `404.php`.

**Review by urchykli — CHANGES_REQUESTED** [review]
Thanks for jumping in on this, Kai!  Just a few things to fix below and inline.
- Add background color (note: on the smallest size screens, you'll need to use `full-width-background` and then remove it on the `small` breakpoint.).
- Remove unused css.
- Link styling (include hover and focus).

File: wp-content/themes/csisnuclearnetwork/assets/_scss/pages/404.scss (hunk 2 above)
Comment (KaiElwood) [reply wp-content/themes/csisnuclearnetwork/assets/_scss/pages/404.scss]: Completed!

**Review by urchykli — APPROVED** [review]

**Pull request merged**